2009-flu-uk.qmd

```diff
@@ -1 +1,437 @@
+
+We assume that the contact rate is constant outside of the holiday period, and that it drops by 30% during the summer and 15% during the half-term. We have thus:
+
+$$\beta(t) = R_{0}*\sigma * h(t)$$
+
+with $h(t)$ a piece-wise constant function equal to 0.7 during the summer, 0.85 during the half-term and 1 otherwise.
+
+To seed the model we assume that a small proportion of the population is infected - equally shared between the E and I compartments. The initial conditions are thus:
+$$
+\begin{aligned}
+    S(0) &= (1 - 2 \alpha) N \\
+    E(0) &= \alpha N \\
+    I(0) &= \alpha N \\
+    R(0) &= 0 \\
+\end{aligned}
+$$
+
+### Observation model
+
+The fundamental component of our transmission model is the 'I' compartment tracking continuously the number of infectious people in our population (an infection prevalence). However, what we observe through our surveillance system is a fraction of the cumulative incidence, more precisely an estimate of the number of new cases in a period of 7 days (a week).
+
+In our `odin` model we can track the weekly cumulative incidence $Z(t)$ by:
+
+1. adding a differential equation integrating the instantaneous incidence:
+$$\frac{dZ}{dt} = \gamma E$$
+
+2. reseting $Z(t) = 0$ at the beginning of each week to only account for the new cases from the current week.
+
+Then we get for each $t$ that is a multiple of 7 (i.e. when we change week):
+
+$$Z(t) = \int_{t-7}^{t} \gamma E(s) \ ds$$
+i.e. the cumulative incidence over a week.
+
+Finally, we need to account for the imperfection of the surveillance system, assuming that only a fraction $\rho$ of the infections are captured by the surveillance system and accounting for potential overdispersion we define the following observation model using a Negative-Binomial distribution with mean $\mu = \rho Z(t)$ and size $\eta$:
+$$
+\text{Cases(t)} \sim \text{NegativeBinomial}(\mu = \rho Z(t), \text{size} = \eta),
+$$
+where $\rho$ is the ascertainment probability, and $\eta$ is the dispersion parameter.
+
+::: {.callout-note}
+Even if the surveillance system was perfect we would expect a difference between the number of *symptomatic* cases and the number of infections as a proportion of flu cases are *asymptomatic* or 'pauci-symptomatic' (showing very few symptoms). 
+:::
+
+### Summary of model parameters
+
+A summary of the parameters of our model:
+
+- $R_{0}$, the basic reproduction number,
+- $\gamma$, the inverse of the mean incubation period set to 1 day,
+- $\sigma$, the recovery rate, the inverse of the mean recovery time assumed to be 1.25 days,
+- $h(t)$, the reduction of contact due to holidays, assumed piecewise constant,
+- $\rho$, the ascertainment probability i.e. the proportion of infections we expect to ascert as cases,
+- $\eta$, the size parameter of our Negative Binomial observation model,
+- $N$ the total size of the population in England and Wales in 2009.
```

Comment:
maybe explain here which parameters you will treat as known and which as unknown

Reply:
Done